data/gegno/gegno power.txt

```diff
@@ -0,0 +1,134 @@
+	mass 23
+	thumbnail outfit/gbatterysmall
+	"energy capacity" 2237
+	"outfit space" -23
+	description "Battlezone batteries are primarily used on the Gegno's medium-range combat vessels, holding a decent amount of power for long lasting engagements. They also fit nicely on larger vessels as an extra power source to feed their larger systems."
+
+outfit "R03 Warforge Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "R03 Warforge Batteries"
+	cost 61700
+	mass 41
+	thumbnail outfit/gbatterymed
+	"energy capacity" 4469
+	"outfit space" -41
+	description "Warforge batteries have proved their usefulness over the years as large, sturdy battery packs. They support the equally large volleys that are fired from Gegno ships, and help maintain an excess of power for engine usage."
+
+outfit "R04 Crusade Battery"
+	category Power
+	licenses
+		"Gegno Civilian"
+		"Vi Evocati"
+		"Vi Centurion"
+		"Vi Lord"
+	plural "R04 Crusade Batteries"
+	cost 136000
+	mass 83
+	thumbnail outfit/gbatterylarge
+	"energy capacity" 8885
+	"outfit space" -83
+	description "As the largest battery mounted on Gegno ships, the Crusade Battery supplies almost as much power as a small generator, enough to support the massive warships of both the Scin and the Vi. However, the Gegno are inefficient at increasing density with size, as done with human batteries, so these fall behind similarly sized options."
+
+outfit "C3 Scrap Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C3 Scrap Cells"
+	cost 54000
+	mass 25
+	thumbnail outfit/vicoretiny
+	"energy generation" 1.4
+	"heat generation" 2.2
+	"outfit space" -25
+	description "The Scrap Cell is a small radiothermal generator that produces energy generation in a pinch, but its heat production can affect the Gegno's slightly hotter running tech. Its output is very limited, and is only used in tandem with the Gegno's smaller-sized systems."
+
+outfit "C7 Brawl Cell"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C7 Brawl Cells"
+	cost 68100
+	mass 39
+	thumbnail outfit/vicoresmall
+	"energy generation" 2.5
+	"heat generation" 3.9
+	"outfit space" -39
+	description "C7 Brawl Cells are a slight deviation from the other C class cores as it is set upright-forward, a more rudimentary configuration. The tried and tested design, however, still serves the Gegno well, though it will likely be phased out when they are better able to scale up newer technologies."
+
+outfit "C17 Warzone Core"
+	category Power
+	licenses
+		"Gegno Civilian"
+	plural "C17 Warzone Cores"
+	cost 2472000
+	mass 87
+	thumbnail outfit/vicoremed
+	"energy generation" 14.1
+	"heat generation" 38.3
+	"outfit space" -87
+	description "As did humans, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks even more discomforting than a human one."
```

Comment:
Humanity didn't really develop nuclear tech because their cities "required" it.
```suggestion
	description "Like humanity, the Gegno began developing nuclear technology once their cities grew massive enough to require it. The Warzone Core is a medium-sized nuclear fission reactor that is a suitable power source for most of the Gegno's large ships, though it looks more disconcerting than a human reactor."
```

Reply:
Instead of required, something more like, another means of efficient power generation?